apps/admin_app/mix.exs

```diff
@@ -46,6 +46,8 @@ defmodule AdminApp.Mixfile do
       {:snitch_core, "~> 0.0.1", in_umbrella: true},
       {:guardian, "~> 1.0"},
       {:params, "~> 2.0"},
+      {:quantum, "~> 2.3"},
+      {:timex, "~> 3.0"},
```

Comment:
Quantum has `timex` dependency in it. We can remove this unless we want to override the dependency.